CHANGELOG.md

```diff
@@ -33,14 +33,18 @@
   [SimplyDanny](https://github.com/SimplyDanny)
   [#5952](https://github.com/realm/SwiftLint/issues/5952)
 
+* Remove lints for `redundant_sendable` on protocols, where it is not redundant.
+  [riley-williams](https://github.com/riley-williams)
+  [#5958](https://github.com/realm/SwiftLint/issues/5958)
+
 ## 0.58.0: New Year’s Fresh Fold
 
 #### Breaking
 
 * The command plugin now requires write permissions so that it works with the `--fix` option without an error.  
   [SimplyDanny](https://github.com/SimplyDanny)
 
-* The artifact bundle name has changed. `SwiftLintBinary-macos.artifactbundle.zip` is now called 
+* The artifact bundle name has changed. `SwiftLintBinary-macos.artifactbundle.zip` is now called
```

Comment:
All these changes need to be reverted. I guess your editor did just silently clean them up ...

Reply:
Ooooh boy sorry, I didn't double check this diff. Ironic considering the repo haha

Reply:
LOL sure if you actually want these I'm happy to open another PR

Reply:
Yes please!